tpcx_bb/cluster_configuration/cluster-startup.sh

```diff
@@ -0,0 +1,63 @@
+#IB, NVLINK, or TCP
+CLUSTER_MODE=$1
+
+# TODO: Unify interface/IP setting/getting for cluster startup
+# and scheduler file
+INTERFACE="enp134s0f1"
+
+MAX_SYSTEM_MEMORY=$(free -m | awk '/^Mem:/{print $2}')M
+DEVICE_MEMORY_LIMIT="25GB"
+POOL_SIZE="30GB"
+
+# Fill in your environment name and conda path on each node
+CONDA_ENV_NAME="rapids-tpcxbb-bsql-20200701-1501"
+CONDA_ENV_PATH="/raid/nicholasb/miniconda3/etc/profile.d/conda.sh"
+
+# TODO: Remove hard-coding of scheduler
+SCHEDULER="exp01"
+LOGDIR="/tmp/tpcx-bb-dask-logs/"
+WORKER_DIR="/tmp/tpcx-bb-dask-workers/"
+
+# Purge Dask worker and log directories
+rm -rf $LOGDIR/*
+mkdir -p $LOGDIR
+rm -rf $WORKER_DIR/*
+mkdir -p $WORKER_DIR
+
+# Purge Dask config directories
+rm -rf ~/.config/dask
+
+# Activate conda environment
+source $CONDA_ENV_PATH
+conda activate $CONDA_ENV_NAME
+
+# Dask/distributed configuration
+export DASK_DISTRIBUTED__COMM__TIMEOUTS__CONNECT="100s"
+export DASK_DISTRIBUTED__COMM__TIMEOUTS__TCP="600s"
+export DASK_DISTRIBUTED__COMM__RETRY__DELAY__MIN="1s"
+export DASK_DISTRIBUTED__COMM__RETRY__DELAY__MAX="60s"
+export DASK_DISTRIBUTED__SCHEDULER__WORK_STEALING="True"
+
+
+# Setup scheduler
+if [ "$HOSTNAME" = $SCHEDULER ]; then
+  # add rapids conda env to jupyterlab
+  python -m ipykernel install --user
+
+  if [ "$CLUSTER_MODE" = "NVLINK" ]; then
+     CUDA_VISIBLE_DEVICES='0' DASK_UCX__CUDA_COPY=True DASK_UCX__TCP=True DASK_UCX__NVLINK=True DASK_UCX__INFINIBAND=False DASK_UCX__RDMACM=False nohup dask-scheduler --dashboard-address 8787 --interface $INTERFACE --protocol ucx > $LOGDIR/scheduler.log 2>&1 &
+  fi
+fi
+
+# If node has a file indicating it should only use a subset of GPUs
```

Comment:
We should remove this section as well.

Reply:
👍

Reply:
Resolved by https://github.com/rapidsai/tpcx-bb/pull/11/commits/2db128380a6e149c944e34deb3bc51efcde30d03